### ISSUE TITLE
Some confusion about parameters and loss functions

### ISSUE THREAD

**Issue by vanity1129** [wangkiw/ICLR23-MEMO#1]
Thanks for your excellent work!

But I have some confusion about the params "**`-ms 3312`**", can you explain why you chose this parameter size?
And about the loss function "**`loss_aux=F.cross_entropy(aux_logits,aux_targets)`**", I noted the definition of the FC layer, "**`self.aux_fc=self.generate_fc(self.out_dim, new_task_size+1)`**", can you provide more explanation about aux_fc?

Many thanks and looking forward to the reply

**Comment by wangkiw** [comment]
Hi, Thank you for your attention. 

- The concrete memory size we use can refer to the supplementary material of our paper.
- For the loss function, we inherit it from DER. It takes all old classes as an auxiliary class.

**Comment by vanity1129** [comment]
Thanks for your reply!
